src/index.js

```diff
@@ -188,6 +196,12 @@ export default {
             capabilities.localIdentifier = connector.connectorInstance.localIdentifierFlag;
             capabilities.local           = true;
         }
+        else {
+            // Can utilise benefit of already running binary on
+            // the system
+            capabilities.local = true;
```

Comment:
This flag is duplicated, could we avoid duplicating it somehow?

Reply:
Moved